agent.py

```
@@ -558,83 +540,220 @@ def dynamic_edge_threshold(yes_buy: float, yes_sell: float) -> float:
+        return None
 
-# ── LLM Fair Probability ──────────────────────────────────
+    except Exception as e:
+        print(f"  detailed_confirm error: {e}")
+        return None
+
+
+# ── Legacy LLM Fair Probability (fallback) ─────────────────
 def openai_fair_prob(
     title: str,
     yes_buy: float,
```

Ruff (0.15.0)

[warning] Unused function argument: `yes_buy`

(ARG001)